### PR TITLE
Stardew valley: Create Option Groups

### PR DESCRIPTION
**Pull request by agilbert1412**
## What is this fixing or adding?
Created Option groups for Stardew Valley, to improve the user experience

## How was this tested?
Ran the webhost locally and looked at it. It's alright

## If this makes graphical changes, please attach screenshots.
image

### CODE REVIEW

**Review by benny-dreamly — APPROVED** [review]
I didn’t look at the code much but the visuals look great

File: worlds/stardew_valley/option_groups.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        QuestLocations,
+        Fishsanity,
+        Museumsanity,
+        Friendsanity,
+        FriendsanityHeartSize,
+        Monstersanity,
+        Shipsanity,
+        Cooksanity,
+        Chefsanity,
+        Craftsanity
```
Comment (Exempt-Medic) [inline worlds/stardew_valley/option_groups.py:43]:
```suggestion
        Craftsanity,
```

File: worlds/stardew_valley/option_groups.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    BackpackProgression, ToolProgression, ElevatorProgression, SkillProgression, BuildingProgression, FestivalLocations, \
+    ArcadeMachineLocations, \
+    SpecialOrderLocations, QuestLocations, Fishsanity, Museumsanity, Friendsanity, FriendsanityHeartSize, \
+    NumberOfMovementBuffs, NumberOfLuckBuffs, \
+    ExcludeGingerIsland, TrapItems, MultipleDaySleepEnabled, MultipleDaySleepCost, ExperienceMultiplier, \
+    FriendshipMultiplier, DebrisMultiplier, QuickStart, \
+    Gifting, FarmType, Monstersanity, Shipsanity, Cooksanity, Chefsanity, Craftsanity, Mods
+
+sv_option_groups = [
+    OptionGroup('General', [
```
Comment (Exempt-Medic) [inline worlds/stardew_valley/option_groups.py:13]: These should use double quotes

**Review by Ishigh1 — APPROVED** [review]
Except Multiple Day Sleep being separated on two distinct columns, everything looks good

**Review by NewSoupVi — APPROVED** [review]

**Pull request merged**